javascript-methods/main.js

```diff
@@ -0,0 +1,57 @@
+var first = 1;
+var second = 2;
+var third = 3;
+
+var maximumValue = Math.max(first, second, third);
+console.log('maximumValue: ', maximumValue);
+
+var heroes = ['Superman', 'Batman', 'Spiderman', 'Invincible'];
+
+var randomNumber = Math.random(heroes);
```

Comment:
The `random` method of the `Math` object doesn't take any arguments: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random